azure_monitor/src/azure_monitor/sdk/auto_collection/__init__.py

```diff
@@ -30,7 +35,14 @@ class AutoCollection:
         labels: Dictionary of labels
     """
 
-    def __init__(self, meter: Meter, labels: Dict[str, str]):
+    def __init__(
+        self,
+        meter: Meter,
+        labels: Dict[str, str],
+        span_processor: AzureMetricsSpanProcessor,
```

Comment:
So we expect a user to pass in an exporter into the spanprocessor when instantiating?

Reply:
SpanProcessor doesn't have any reference to an exporter anymore